README.md

```diff
@@ -32,9 +32,16 @@ const publication = await liClient.getPublication({documentId: 1})
 
 We assume that you used the standard signup flow. This would give you a document with id 1. Of course you can change this id to any document in your project.
 
-4. OPTIONAL: Configure an image service (If you define nothing, the default settings will be used)
-
+4. Get a design
+```js
+const design = await liClient.getDesign({name: 'living-times', version: '0.0.14'})
 ```
+
+This loads our magazine example design from the Livingdocs server. You can of course also use your own local design.
+
+5. OPTIONAL: Configure an image service (If you define nothing, the default settings will be used)
+
+```js
```

Comment:
I'd vote for using the default in the intro example here and moving the configuration of an image service further down into the reference doc.